flake.nix

```diff
@@ -33,10 +33,17 @@
           crypto3-debug-tests = (pkgs.callPackage ./crypto3.nix {
             enableDebug = true;
             runTests = true;
+            sanitize = true;
           });
           crypto3-sanitize = (pkgs.callPackage ./crypto3.nix {
-            enableDebug = true;
             runTests = true;
+            enableDebug = false;
+            sanitize = true;
+          });
+          crypto3-clang-sanitize = (pkgs.callPackage ./crypto3.nix {
```

Comment:
maybe just `crypto3-clang-sanitize` enough?

Reply:
You mean, we don't need to check
parallel-crypto3-clang-sanitize and proof-producer-sanitize ? 

Why?

Reply:
corrected